osu.Game.Rulesets.Osu/Edit/PreciseScalePopover.cs

```diff
@@ -140,7 +140,11 @@ protected override void LoadComplete()
             base.LoadComplete();
 
             ScheduleAfterChildren(() => scaleInput.TakeFocus());
-            scaleInput.Current.BindValueChanged(scale => scaleInfo.Value = scaleInfo.Value with { Scale = scale.NewValue });
+            scaleInput.Current.BindValueChanged(scale =>
+            {
+                if (scaleHandler.OperationInProgress.Value)
```

Comment:
similarly as above, should be moved to

https://github.com/ppy/osu/blob/c26e669fc534ef2b292effa0e9aebb9dd714632c/osu.Game.Rulesets.Osu/Edit/PreciseScalePopover.cs#L224